extensions/notification/queueProcessor/QueueProcessor.js

```diff
@@ -158,59 +177,69 @@ class QueueProcessor extends EventEmitter {
      * @param {function} done - callback function
      * @return {undefined}
      */
-    async processKafkaEntry(kafkaEntry, done) {
```

Comment:
why are you making this not async?

Reply:
This PR originally had one commit that only dealt with the environment variable issue

The other commits are from a PR that was merged and that this PR was based on top of, so most of your comments are for those other commits actually 😅

I rebased the branch, so there shouldn't be any confusion anymore